constructor/fcp.py

```diff
@@ -220,7 +221,7 @@ def main(info, verbose=True, dry_run=False):
     version = info["version"]
     download_dir = info["_download_dir"]
     platform = info["_platform"]
-    channel_urls = info.get("channels", ())
+    channel_urls = all_channel_urls(info.get("channels", ()), subdirs=[platform])
```

Comment:
Deliberately leaving out noarch here.

Reply:
I guess we're going to have to be more sophisticated here if we want to be able to include `noarch: generic`, but exclude `noarch: python`.